packaging/installer/UPDATE.md

```diff
@@ -31,6 +31,22 @@ Starting with netdata v1.33.0, you can use Netdata itself to determine the insta
 netdata -W buildinfo | grep -E 'Installation Type|Install type:'
 ```
 
+The following table gives you an overview of the possible _Installation types_ you may bump into:
+
+| Installation-type    | Description                                                                                                                                           |
+|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| binpkg-rpm           | RPM based native packages shipped from Netdata's repos.                                                                                               |
+| binpkg-deb           | DEB based native packages shipped from Netdata's repos.                                                                                               |
+| kickstart-build      | Build from source with the kickstart script's `--build-only` option.                                                                                  |
+| kickstart-static     | Installed the static builds, shipped from netdata via the kickstart script's (option: `--static-only`).                                               |
+| manual-static-ARCH   | Manually installed the Agent static binaries by fetching our tarballs and manually install them. (Offline installation are part of this category)     |
+| legacy-build         | Used for pre-existing kickstart.sh or netdata-installer.sh installations. This exist because we cannot determine how the install originally happened. |
+| legacy-static        | Same as legacy-build, but for static installs.                                                                                                        |
+| oci                  | Installed using our official Docker images, though not necessarily running on Docker                                                                  |
+| custom               | Anything not covered by the other identifiers, including manual builds, manually running netdata-installer.sh, and third-party packages (community).  |
+| Unknown              | Same as `custom`                                                                                                                                      |
```

Comment:
```suggestion
The following table contains all possible installation types:

| Installation-type  | Description                                                                                                                                                 |
|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
| binpkg-rpm         | RPM-based native packages shipped from Netdata's repos.                                                                                                     |
| binpkg-deb         | DEB-based native packages shipped from Netdata's repos.                                                                                                     |
| kickstart-build    | Build from source with the kickstart script's `--build-only` option.                                                                                        |
| kickstart-static   | Installed the static builds, shipped from netdata via the kickstart script's (option: `--static-only`).                                                     |
| manual-static-ARCH | Manually installed static Agent binaries by downloading archives from GitHub and installing them manually. Offline installations are part of this category. |
| legacy-build       | Used for pre-existing kickstart.sh or netdata-installer.sh installations. This exist because we cannot determine how the install originally happened.       |
| legacy-static      | Same as legacy-build, but for static installs.                                                                                                              |
| oci                | Installed using official Docker images from Netdata, though not necessarily running on Docker                                                               |
| custom             | Anything not covered by the other identifiers, including manual builds, manually running netdata-installer.sh, and third-party packages (community).        |
| Unknown            | Same as custom.                                                                                                                                             |
```

Reply:
Slightly change wording (removed "ours").